client/signup/site-mockup/index.jsx

```diff
@@ -0,0 +1,123 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { getSiteTitle } from 'state/signup/steps/site-title/selectors';
+import { getSignupStepsSiteTopic } from 'state/signup/steps/site-topic/selectors';
+import { getVerticalData } from './mock-data';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class SiteMockup extends Component {
+	static propTypes = {
+		title: PropTypes.string,
+		tagline: PropTypes.string,
+		address: PropTypes.string,
+		phone: PropTypes.string,
+		vertical: PropTypes.string,
+	};
+
+	getMockupChromeDesktop() {
+		return (
+			<div className="site-mockup__chrome-desktop">
+				<svg width="38" height="10">
+					<g>
+						<rect width="10" height="10" rx="5" />
+						<rect x="14" width="10" height="10" rx="5" />
+						<rect x="28" width="10" height="10" rx="5" />
+					</g>
+				</svg>
+				<span className="site-mockup__chrome-label">{ translate( 'Website Preview' ) }</span>
+			</div>
+		);
+	}
+
+	getMockupChromeMobile() {
+		return (
+			<div className="site-mockup__chrome-mobile">
+				<span className="site-mockup__chrome-label">
+					{ translate( 'Phone', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Phone', { context: 'Domain Edit Contact Info form.'} )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-mockup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?